config/identical-files.json

```diff
@@ -1,58 +1,58 @@
 {
     "C++ IR Instruction": [
-        "cpp/ql/src/semmle/code/cpp/ir/internal/Instruction.qll",
-        "cpp/ql/src/semmle/code/cpp/ssa/internal/ssa/Instruction.qll",
-        "cpp/ql/src/semmle/code/cpp/ssa/internal/aliased_ssa/Instruction.qll"
+        "cpp/ql/src/semmle/code/cpp/ir/internal/raw_ir/Instruction.qll",
+        "cpp/ql/src/semmle/code/cpp/ir/internal/ssa_ir/Instruction.qll",
+        "cpp/ql/src/semmle/code/cpp/ir/internal/aliased_ssa_ir/Instruction.qll"
```

Comment:
Isn't the `_ir` suffix on the directories redundant? They're already below a directory named `ir`.